hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -1618,15 +1621,13 @@ public void testGetTrashRoots() throws IOException {
    * since fs.rename(src,dst,options) is enabled.
    */
   @Test
-  @Flaky("HDDS-6646")
   public void testRenameToTrashEnabled() throws Exception {
```

Comment:
This particular test never calls deleteRootDir(), not sure how fixing deletRootDir makes  this test non-flaky, Anyway I feel this functionality is tested in testTrash method in the same class in much detail . This test was written before the trash feature was complete, I feel we can remove this test from the class.

Reply:
Thanks Sadanand for review. As discussed, we need to test Trash related tests in separate Test class spinning separate mini cluster having  emptier thread started with close match of timings of relevant  assertion statements and corresponding trash and checkpoint interval. So I have moved assertion for above test in testTrash test case and will be handled as part of [HDDS-6645](https://issues.apache.org/jira/browse/HDDS-6645)

Reply:
Thanks Devesh, sounds good to me.